packages/mdx/test/compile.js

```diff
@@ -792,8 +792,8 @@ test('jsx', async () => {
       '/*@jsxRuntime automatic @jsxImportSource react*/',
       'function _createMdxContent(props) {',
       '  const _components = Object.assign({',
-      '    p: "p",',
-      '    em: "em"',
+      '    em: "em",',
+      '    p: "p"',
       '  }, props.components);',
```

Comment:
Is really only this one test case affected? I’d imagine a ton more would change?

Reply:
Yes. I was surprised too!